website/docs/r/container_registry.markdown

```diff
@@ -0,0 +1,54 @@
+---
+layout: "scaleway"
+page_title: "Scaleway: scaleway_container_registry"
+description: |-
+  Manages Scaleway Container Registries.
+---
+
+# scaleway_container_registry
+
+Creates and manages Scaleway Container Registry. For more information see [the documentation](https://developers.scaleway.com/en/products/registry/api/).
+
+## Examples
+
+### Basic
+
+```hcl
+resource "scaleway_container_registry" "main" {
+    region = "fr-par"
+    name = "main_cr"
+    description = "Main container registry"
+    is_public = false
+}
+```
+
+## Arguments Reference
```

Comment:
Can you move `region` and `organization_id` arguments after `is_public`? The majority of users will use the default ones.